Library/Homebrew/test/utils/analytics_spec.rb

```diff
@@ -0,0 +1,93 @@
+require "utils/analytics"
+
+describe Utils::Analytics do
+  describe "::os_prefix_ci" do
+    context "when anonymous_os_prefix_ci is not set" do
+      it "returns OS_VERSION and prefix when HOMEBREW_PREFIX is not /usr/local" do
+        expect(described_class.os_prefix_ci).to include("#{OS_VERSION}, non-/usr/local")
+      end
+
+      it "includes CI when ENV['CI'] is set" do
+        CI = ENV["CI"]
+        ENV["CI"] = "true"
+
+        expect(described_class.os_prefix_ci).to include("CI")
+
+        ENV["CI"] = CI
```

Comment:
Can you put this in an `ensure`? If an exception were raised in this test there's a chance that we would skip this, affecting the results of other tests.